app/models/concerns/node_shared.rb

```diff
@@ -1,5 +1,6 @@
 module NodeShared
   extend ActiveSupport::Concern
+  include ActionView::Helpers::ApplicationHelper
```

Comment:
Here I tried getting us access to the application helpers, and esp the line with the `translation()` helper:

https://github.com/publiclab/plots2/blob/main/app/helpers/application_helper.rb#L157

Reply:
Looks like that didn't work -- 

```
NodeSharedTest#test_that_NodeShared_can_be_used_to_convert_short_codes_like_[map:people:___:___]_into_maps_which_display_peoples_locations (19.66s)
Minitest::UnexpectedError:         ActionView::Template::Error: undefined method `translation' for #<#<Class:0x0000563ce4bb9b90>:0x0000563ce58d0018>
```

Reply:
```suggestion
```